src/test/java/ch/uzh/ifi/hase/soprafs22/service/UserServiceTest.java

```diff
@@ -54,6 +54,7 @@ public void setup() {
     testUser.setUsername("testUsername");
     testUser.setPassword("1234");
     testUser.setGender(Gender.OTHER);
+    testUser.setBirthday(new Date());
```

Comment:
Why do you set the birthday as today (would make more sense to have him be older than 18)